plugins/source/aws/resources/services/lambda/functions.go

```diff
@@ -31,24 +30,38 @@ func Functions() *schema.Table {
 				Name:     "arn",
 				Type:     schema.TypeString,
 				Resolver: schema.PathResolver("Configuration.FunctionArn"),
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
 			},
 			{
 				Name: "policy_revision_id",
 				Type: schema.TypeString,
 			},
 			{
-				Name: "policy_document",
-				Type: schema.TypeJSON,
+				Name:     "policy_document",
+				Type:     schema.TypeJSON,
+				Resolver: resolveResourcePolicy,
 			},
 			{
-				Name: "code_signing_config",
-				Type: schema.TypeJSON,
+				Name:     "code_signing_config",
+				Type:     schema.TypeJSON,
+				Resolver: resolveCodeSigningConfig,
 			},
 			{
 				Name:     "code_repository_type",
 				Type:     schema.TypeString,
 				Resolver: schema.PathResolver("Code.RepositoryType"),
 			},
+			{
+				Name: "update_runtime_on",
+				Type: schema.TypeString,
+			},
```

Comment:
Add a comment about missing resolver and how it's managed by `resolveRuntimeManagementConfig`?